test/kubemark/run-e2e-tests.sh

```diff
@@ -32,6 +32,12 @@ export KUBE_MASTER_URL="https://${KUBE_MASTER_IP}"
 export KUBECONFIG="${ABSOLUTE_ROOT}/test/kubemark/resources/kubeconfig.kubemark"
 export E2E_MIN_STARTUP_PODS=0
 
+: ${KUBE_ENABLE_CLUSTER_AUTOSCALER:="false"}
```

Comment:
Move this variable to `cluster/kubemark/gce/config-default.sh`. It's a useful option to be made visible.
Also rename it to sth like ENABLE_KUBEMARK_AUTOSCALER to not clash with the variable for the base cluster.